### PR TITLE
Ensures that age range filters are symbols

### PR DESCRIPTION
**Pull request by kwiliarty**

### CODE REVIEW

File: app/models/concerns/filter/filter_scopes.rb [hunk 1]
```diff
@@ -60,20 +60,22 @@ module Filter::FilterScopes
     end
 
     private def filter_for_age(scope)
-      return scope unless @filter.age_ranges.present? && (@filter.available_age_ranges.values & @filter.age_ranges).present?
+      # The age ranges will be strings if we had saved them in the db, so we map to symbols
+      age_ranges = @filter.age_ranges.map(&:to_sym)
```
Comment (eanders) [inline app/models/concerns/filter/filter_scopes.rb]: Let's throw a select present in here
```suggestion
      age_ranges = @filter.age_ranges.select(:present?).map(&:to_sym)
```

**Review by eanders — CHANGES_REQUESTED** [review]
I think it's worth throwing out blanks, either by selecting present or anding it with available options.

File: app/models/concerns/filter/filter_scopes.rb (hunk 1 above)
Comment (kwiliarty) [reply app/models/concerns/filter/filter_scopes.rb]: ✅

**Comment by kwiliarty** [comment]
All set.

**Review by eanders — APPROVED** [review]

**Comment by eanders** [comment]
No longer needed, #1731 makes this obsolete.

**Pull request closed without merging**